### PR TITLE
Revisit cookie handling

### PR DESCRIPTION
**Pull request by MrGoumX**

### CODE REVIEW

File: src/session.coffee [hunk 1]
```diff
@@ -84,7 +84,10 @@ define [
         [null, null]
 
     _getMetadataFromCookie: ->
-      Biskoto.get(Settings.cookies[@cookie_policy].meta.name)
+      if data = Biskoto.get(Settings.cookies.full.meta.name)
```
Comment (alexunder193) [inline src/session.coffee:87]: Is there any difference between the previous code and the current?

Comment (alexunder193) [reply src/session.coffee:87]: discussed offline

**Review by harrisred — APPROVED** [review]
LGTM 👍

**Review by mrwhizzy — APPROVED** [review]
+1 LGTM! :shipit:

**Pull request merged**